sw/sw.py

```diff
@@ -113,6 +113,8 @@ async def person(self, ctx, person_id: Union[int,str]):
                 if response.status == 404:
                     return await ctx.send("Invalid Person ID.")
```

Comment:
Since this is a string/query and not an ID per se, could this and the other ones in the string queries be converted to just "Invalid person/film/etc"?